CMakeLists.txt

```diff
@@ -99,16 +99,23 @@ set(IGN_PHYSICS_VER ${ignition-physics5_VERSION_MAJOR})
 #--------------------------------------
 # Find ignition-sensors
 ign_find_package(ignition-sensors6 REQUIRED
+  # component order is important
```

Comment:
I can see this [warning](https://build.osrfoundation.org/job/ignition_gazebo-ci-pr_any-ubuntu_auto-amd64/6782/cmake/) on Ubuntu 

```
CMake 3.11.0 or higher is required. You are running version 3.10.2
```

Reply:
ebe8bdf